tools/geospatial.go

```diff
@@ -22,16 +23,19 @@ type GeoData struct {
 
 // Features ...
 type Features struct {
-	Rivers              []VectorLayer
-	XS                  []VectorLayer
-	Banks               []VectorLayer
-	StorageAreas        []VectorLayer
-	TwoDAreas           []VectorLayer
-	HydraulicStructures []VectorLayer
+	Rivers              []VectorFeature
```

Comment:
Nice. This is certainly  a feature and not a layer. Have you run any tests to ensure this has been refactored elsewhere in the code?

Reply:
I fixed it everywhere and ran the API on a few test cases to ensure nothing had broken.